sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/datastore/V1Beta3.java

```diff
@@ -694,141 +752,170 @@ public void populateDisplayData(DisplayData.Builder builder) {
       super.populateDisplayData(builder);
       builder
           .addIfNotNull(DisplayData.item("projectId", projectId)
-              .withLabel("Output Project"));
+              .withLabel("Output Project"))
+          .addIfNotNull(DisplayData.item("mutationFn", mutationFn.getClass().getName())
+              .withLabel("Datastore Mutation Function"));
```

Comment:
Add the delegate data from the `mutationFn`. (`SimpleFunction` implements `HasDisplayData`).